core/controllers/access_validators_test.py

```diff
@@ -224,16 +224,16 @@ class BlogHomePageAccessValidationHandlerTests(test_utils.GenericTestBase):
     """Checks the access to the blog home page and its rendering."""
 
     def test_blog_home_page_access_without_logging_in(self):
-        self.get_json(
+        self.get_html_response(
```

Comment:
I'm a bit confused by these tests. Shouldn't they require the feature flag to be enabled before the pages can be accessible? There should also be tests to check correct behaviour when the feature flag is turned off.

Reply:
We check for feature flag in the ..page-root.component and then accordingly fetch response from backend and validate access according to new angular router structure.